app/Helpers/database/release.php

```diff
@@ -10,94 +10,20 @@
  * https://github.com/RetroAchievements/rcheevos/blob/develop/src/rcheevos/consoleinfo.c
  * https://github.com/RetroAchievements/rcheevos/blob/develop/test/rcheevos/test_consoleinfo.c
  */
+function getValidConsoleIds(): array
+{
+    return Cache::store('array')->rememberForever('system:active-ids', function () {
+        return System::active()->pluck('ID')->toArray();
+    });
```

Comment:
Keep a list of valid console IDs in each request. Fetching the data to redis wasn't any more efficient than fetching it from the database.